tests/test_buffer_dynamic.py

```diff
@@ -778,3 +779,73 @@ def test_bufferPortMaxParameter(self, dvs, testlog):
         assert int(fvs["max_queues"]) and int(fvs["max_priority_groups"])
 
         self.cleanup_db(dvs)
+
+    def test_startBufferManagerWithoutLosslessParameter(self, dvs, testlog):
+        buffer_model.disable_dynamic_buffer(dvs.get_config_db(), dvs.runcmd)
+        buffer_model.enable_dynamic_buffer(dvs.get_config_db(), dvs.runcmd, False)
+        time.sleep(5)
+
+        self.setup_db(dvs)
+
+        expectedProfile = self.make_lossless_profile_name(self.originalSpeed, self.originalCableLen)
+
+        # Startup interface
+        dvs.port_admin_set('Ethernet0', 'up')
+
+        # Configure lossless PG 3-4 on interface
+        self.config_db.update_entry('BUFFER_PG', 'Ethernet0|3-4', {'profile': 'NULL'})
+
+        # No lossless profile created
+        time.sleep(5)
+        self.app_db.wait_for_deleted_entry('BUFFER_PROFILE_TABLE', expectedProfile)
+
+        # Initialize lossless parameters
+        buffer_model.initialize_lossless_parameters(dvs.get_config_db(), dvs.runcmd)
```

Comment:
can you add this in a try block? in case there is any failure above and this does not get configured, the rest of the testcases will get affected since 'enable_dynamic_bufer' is a class level fixture

Reply:
I removed this part and replaced it with the mocked test.